extra-files/galaxy-kickstart/galaxy-kickstart_tool_list.yml

```diff
@@ -0,0 +1,37 @@
+tools:
+- name: data_manager_fetch_genome_dbkeys_all_fasta
+  owner: devteam
+  revisions:
+  - 776bb1b478a0
+  tool_panel_section_label: null
```

Comment:
What do you propose:
No tool_panel_section_label at all, or a tool_panel_section_label for data managers ?